spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -200,17 +220,24 @@
         expect(validating_numericality(even: true, greater_than: 18))
           .to matcher.even.is_greater_than(18)
       end
+
       it do
         expect(validating_numericality(odd: true, less_than_or_equal_to: 99))
           .to matcher.odd.is_less_than_or_equal_to(99)
       end
 
       it do
+        expect(validating_numericality(strict: true, greater_than: 99))
+          .to matcher.is_greater_than(99).strict
+      end
+
+      it do
```

Comment:
This test is fine. We are obviously not testing every possible combination here, but, I think this is okay.

I don't know how the style here crept through but it should be something like:

``` ruby
record = validating_numericality(
  only_integer: true,
  greater_than: 18,
  less_than: 99
)
expect(record).to matcher.only_integer.is_greater_than(18).is_less_than(99).strict
```